Tests/SonarQube.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs

```diff
@@ -356,6 +520,15 @@ private static void AddEmptyContentFile(ProjectDescriptor descriptor, string pro
             descriptor.ContentFiles.Add(emptyFilePath);
         }
 
+        private static string CreateEmptyFile(string folder, string extension)
+        {
+            string emptyFilePath = Path.Combine(folder, "empty_" + Guid.NewGuid().ToString() + ".xxx");
+            emptyFilePath = Path.ChangeExtension(emptyFilePath, extension);
```

Comment:
It's a lazy way of not having to check whether the supplied file extension includes the "." or not.
The content of the file isn't important; it's just an easy way to create a file.